src/Compilers/CSharp/Portable/Syntax/SyntaxFactory.cs

```diff
@@ -314,8 +314,8 @@ public static SyntaxToken Identifier(SyntaxTriviaList leading, string text, Synt
         /// Creates a verbatim token with kind IdentifierToken containing the specified text.
         /// </summary>
         /// <param name="leading">A list of trivia immediately preceding the token.</param>
-        /// <param name="text">The raw text of the identifier name, including any escapes or leading '@'
-        /// character as it is in source.</param>
+        /// <param name="text">The name of the identifier, not including any escapes or leading '@'
```

Comment:
"The name of the identifier" seems a little confusing. Perhaps just "The identifier" or "The identifier text" instead.
```suggestion
        /// <param name="text">The identifier, not including any escapes or leading '@'
```